apps/api/src/modules/notifications/dtos/single-notification.response.dto.ts

```diff
@@ -0,0 +1,97 @@
+import { ObjectType, Field } from '@nestjs/graphql';
+import { Column, CreateDateColumn, PrimaryGeneratedColumn, UpdateDateColumn } from 'typeorm';
+import { IsEnum, IsObject, IsOptional } from 'class-validator';
+import { GraphQLJSONObject } from 'graphql-type-json';
+import { ChannelType, DeliveryStatus } from 'src/common/constants/notifications';
+import { Status } from 'src/common/constants/database';
+import { Notification } from '../entities/notification.entity';
+import { ArchivedNotification } from 'src/modules/archived-notifications/entities/archived-notification.entity';
+
+@ObjectType()
+export class SingleNotificationResponse {
```

Comment:
can we not just call it NotificationResponse?

And we should have this class already right?

Reply:
I see that we gave this name already to the query which fetched notifications should have named that as notificationsResponse or something like that.

Okay for now.

We should fix these kind of issues slowly.